src/vuestic-theme/vuestic-components/va-rating/VaRating.demo.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <VbDemo>
+    <VbContainer>
+      <table class="table table-bordered">
+        <tr>
+          <th>Description</th>
+          <th>Vuestic Rating</th>
+        </tr>
+        <tr>
+          <td>Types</td>
+          <td>
+            <va-rating
+              v-model="value"
+            />
+            default
+            <va-rating
```

Comment:
Demo:
![image](https://user-images.githubusercontent.com/5394573/53444682-e01e7380-3a1f-11e9-80ef-55fa2cdc9ce3.png)

Presentation: 
![image](https://user-images.githubusercontent.com/5394573/53444703-ea407200-3a1f-11e9-8fba-afbbae80a872.png)

Design: 
![image](https://user-images.githubusercontent.com/5394573/53444727-fa585180-3a1f-11e9-9d66-270b9fa212a8.png)

I wouldn't call this a subtle difference.